src/algorithms/shell.rs

```diff
@@ -0,0 +1,40 @@
+//! [Shell sort](https://en.wikipedia.org/wiki/Shellsort)
+
+use algorithms::Algorithm;
+use array::Array;
+
+/// [Bubble sort](https://en.wikipedia.org/wiki/Bubble_sort)
```

Comment:
Could you, please, update this link too?

Reply:
I have updated the code, but I am not sure how to add it here